LICENSE-MIT

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Astral Software Inc.
+Copyright (c) 2024 Astral Software Inc.
```

Comment:
Thanks! I wonder if this should be 2025?

Reply:
Oh, I forgot that it is 2025 already